paddle/fluid/framework/new_executor/instruction/control_flow/tuple_push_instruction.cc

```diff
@@ -20,6 +20,37 @@
 
 namespace paddle {
 namespace framework {
+void ParsePlace(pir::Type type, OpFuncType type_) {
```

Comment:
```suggestion
void ParsePlace(pir::Type type, OpFuncType& type_) {
```
这里的type_是值copy，调用此函数后，类对象里的type_成员并不会改变

Reply:
另外建议命名方式也修改下，比如 OpFuncType& op_func_type